lib/madmin/fields/belongs_to.rb

```diff
@@ -15,8 +15,9 @@ def to_param
         "#{attribute_name}_id"
       end
 
-      def index_path
-        associated_resource.index_path(format: :json)
+      def index_path(format: :json)
+        associated_resource.index_path(format: format)
+      rescue NameError
```

Comment:
Word of warning, we recently found a bug at work that was caused by a silent `rescue NameError` I had added and it broke something completely unexpected without telling us about it. Perhaps at least a log message should be emitted?

Reply:
Yeah, good call. I meant to refactor this so `NameError` is only rescued around the `constantize` calls.